src/System.Threading.Channels/pkg/System.Threading.Channels.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\src\System.Threading.Channels.csproj">
-      <SupportedFramework>netcoreapp2.0;net461;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>net46;net45;netcore45;netcoreapp2.0;netcoreapp1.0;wpa81;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
Why did you put net45, netcore45, and wpa81? I suspect that is what is causing your validation errors.

Reply:
ungh